src/unstract/sdk/file_storage/helper.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+import fsspec
+from google.oauth2 import service_account
+
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage.constants import GCS
+from unstract.sdk.file_storage.fs_provider import FileStorageProvider
+
+logger = logging.getLogger(__name__)
+
+
+class FileStorageHelper:
+    @staticmethod
+    def file_storage_init(provider: FileStorageProvider, credentials: dict()):
+        """Initialises file storage based on provider.
+
+        Args:
+            provider (FileStorageProvider): Provider
+            credentials (dict): Credentials based on provider
+
+        Returns:
+            NA
+        """
+        if provider == FileStorageProvider.GCS:
+            return FileStorageHelper.gcs_init(credentials)
+        elif provider == FileStorageProvider.Local:
+            return FileStorageHelper.local_file_system_init()
+
+    @staticmethod
+    def gcs_init(credentials):
+        """Initialises FileStorage backed up by GCS.
+
+        Args:
+            credentials (dict): credentials
+
+        Returns:
+            NA
+        """
+        try:
+            gcs_creds = service_account.Credentials.from_service_account_file(
+                credentials[GCS.CREDS],
```

Comment:
@gaya3-zipstack this can lead to `KeyError`s if this env is not properly intitialized - where do we validate this `credentials` dict?
NIT: Please add type hints to all methods

Reply:
Yes @chandrasekharan-zipstack  There will be a KeyError that will be caught by the excpetion and printed.